### PR TITLE
Fix usage with raw loader

### PR DESCRIPTION
**Pull request by mistic**
That should fix https://github.com/webpack-contrib/cache-loader/issues/68

/CC @evilebottnawi

### CODE REVIEW

**Comment by malash** [comment]
Could it be better if we use `base64` for raw data encoding to reduce the cache file size ?

https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings

File: test/__snapshots__/cacheContext-option.test.js.snap [hunk 1]
```diff
@@ -22,9 +22,55 @@ Array [
...
+          108,
+          101,
+          95,
+          49,
+          39,
+          41,
+          59,
+          10,
+        ],
+        "type": "Buffer",
```
Comment (malash) [inline test/__snapshots__/cacheContext-option.test.js.snap]: `JSON.parse` won't convert this field to buffer.

https://github.com/webpack-contrib/cache-loader/blob/master/src/index.js#L241

Comment (evilebottnawi) [reply test/__snapshots__/cacheContext-option.test.js.snap]: Yes, you are right

Comment (evilebottnawi) [reply test/__snapshots__/cacheContext-option.test.js.snap]: I think we need improve logic with save

Comment (mistic) [reply test/__snapshots__/cacheContext-option.test.js.snap]: @malash @evilebottnawi I'm now using `json-buffer` to parse and stringify our objects which solves the problem with Buffers and also encodes them with base64 on stringify.

**Comment by mistic** [comment]
@malash thanks for the suggestions. I had them in account on the last changes 👍

**Comment by mistic** [comment]
@evilebottnawi it could have been an excellent solution but after trying that serializer-to-js I found performance penalties. I think we should stick with buffer-json for the time being 😃

**Pull request merged**